spec/system/admins/company_values_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudos' do
+  let(:admin) { create(:admin) }
+
+  before do
```

Comment:
what should happen, if a user is not logged in as an admin? Tests should cover that scenario as well.